src/index.js

```diff
@@ -3,7 +3,7 @@ import React from "react";
 import { defaultProps } from "./default-props";
 import { propTypes } from "./prop-types";
 import MaterialTable from "./material-table";
-import withStyles from "@material-ui/core/styles/withStyles";
+import withStyles from "@material-ui/styles/withStyles";
```

Comment:
I think we should stay the core import to access the theme from the surroundings? https://next.material-ui.com/styles/basics/#using-the-theme-context

Reply:
Yes, I agree that the imports need to be refactored in general. For this PR I tried to keep the changeset minimal towards upstream so I basically just renamed things where needed.

Reply:
Import { withstyles } from '@material-ui/core' works pre [MaterialUI docs](https://material-ui.com/guides/typescript/#usage-of-withstyles)